src/VulkanTutorial/OffscreenRenderingVulkanTutorialMain.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <atomic>
+#include <csignal>
+#include <thread>
+
+#include "VulkanTutorial/OffscreenRenderingVulkanTutorial.h"
+
+namespace {
+std::atomic<bool> exit_requested = false;
+
+void SigintHandler(int signum) {
+  if (signum == SIGINT) {
+    exit_requested = true;
+  }
+}
+
+void InstallSigintHandler() {
```

Comment:
Nit: It would be nice to have a brief comment here, what this does/why this is needed.

Reply:
Done.